test/takeScreenshot.test.js

```diff
@@ -93,6 +115,15 @@ describe("POST /api/cook", () => {
     await compareImage({ imageName, imageBuffer });
   });
 
+  it("should create default image (multipart)", async () => {
+    const imageName = "default";
```

Comment:
there will be a conflict with existing image that is already named like this (test above). I think it should be named differently

Reply:
Ok, my point was that regardless the content type the generated image should be the same.

I created separated images for this tests and other testing custom TTF fonts below.